tools/scripts/build-rust-headers.sh

```diff
@@ -0,0 +1,5 @@
+#!/bin/bash
```

Comment:
Let's make the script fail upon any command exiting with a non-zero status code.

```suggestion
#! /bin/bash
set -e
```